client/components/tinymce/index.jsx

```diff
@@ -496,7 +496,7 @@ module.exports = React.createClass( {
 		} );
 
 		return (
-			<div>
+			<div className="tinymce-container">
```

Comment:
@bisko this caused an eslint error because `jsx-class-namespace` expects the `className` to be `tinymce`, not `tinymce-container`.

I used `tinymce-container`, though, because `tinymce` is already used on the `<textarea>`. Would the best thing to do here be to make a new PR to rename the `<textarea>` to something like `tinymce-raw-text`?